.github/workflows/workflow_update_proposal.md

```diff
@@ -0,0 +1,158 @@
+# Pipeline Update Proposal

```

Comment:
Can you remove this file add this as an issue on the internal repo?

Reply:
Which one is our internal repo? marqoai/marqo?

Reply:
Nvm. Done!